packages/ui/src/components/va-carousel/tests/VaCarousel.spec.ts

```diff
@@ -1,10 +1,22 @@
 import { mount } from '@vue/test-utils'
+import { describe, it, expect } from 'vitest'
 
 import VaCarousel from '../VaCarousel.vue'
+import {
+  GLOBAL_CONFIG,
+  createGlobalConfig,
+} from '../../../services/global-config/global-config'
 
-describe('VaCarousel', () => {
+// TODO Test broken. Getting error.
```

Comment:
Passed for me.

Reply:
@m0ksem double checked, still broken. Can you double check you don't have error?

<img width="914" alt="image" src="https://user-images.githubusercontent.com/5394573/180898565-23bb3891-20c4-47a0-98f8-eb9e2f905d20.png">

Reply:
Hope you don't mind if we handle this after the merge. :)